src/index.html

```diff
@@ -11,6 +11,10 @@
       rel="stylesheet"
       href="./styles/main.scss"
```

Comment:
The stylesheet link is pointing to a SCSS file. Browsers cannot directly interpret SCSS files; they need to be compiled to CSS. Ensure that the SCSS is compiled to a CSS file and the link is updated accordingly.